packages/fake-driver/lib/fake-app.js

```diff
@@ -1,14 +1,13 @@
-import B from 'bluebird';
-import fs from 'fs';
+import {fs} from '@appium/support';
```

Comment:
even though there's not a rule, let's stick with the `{ <name> }` spacing style if we notice it.